### PR TITLE
Better CI error messages

### LINKED ISSUE #378: Better CI error messages
In travis.yml, we define the following testing stages 
```
- ls -lah
- docker build -t sqlflow:dev -f Dockerfile.dev .
- docker run --rm -v $GOPATH/src:/go/src -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow:dev pre-commit run -a
# NOTE: mount all $GOPATH into container, then after build and test,
# $GOPATH/bin will have expected binaries.
- docker run --rm -v $GOPATH:/go -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow:dev bash scripts/test.sh
```
Travis CI considers them as independent stages. For example, in [the CI log](https://travis-ci.com/sql-machine-learning/sqlflow/builds/111532246), pre-commit fails so the CI fails. However, you need to scroll up serveral hundred lines to find the reason.

I would suggest putting all the stages in a single `test.sh` file and make each line dependent via `set -e`.

### PR DESCRIPTION
**Pull request by tonyyang-svail**
Fix #378

### CODE REVIEW

File: scripts/test.sh [hunk 1]
```diff
@@ -3,10 +3,6 @@ set -e
 
 service mysql start
 
-cat example/datasets/popularize_churn.sql | mysql -uroot -proot
```
Comment (tonyyang-svail) [inline scripts/test.sh:6]: `sql/db_test.go` is responsible for data population.

File: scripts/test_runner.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
```
Comment (typhoonzero) [inline scripts/test_runner.sh:2]: Have you tested adding `set -e` in travis.yml? if that works we may reduce the level of script calls.

File: scripts/test.sh (hunk 1 above)
Comment (typhoonzero) [reply scripts/test.sh:6]: I'm not sure `db_test.go` can always run before `main_test` since `main_test` depend on these sample data.

**Comment by tonyyang-svail** [comment]
> I'm not sure db_test.go can always run before main_test since main_test depend on these sample data.

@typhoonzero You are right. I forgot the test in `cmd/` also needs populated data. My bad.

File: scripts/test_runner.sh (hunk 2 above)
Comment (tonyyang-svail) [reply scripts/test_runner.sh:2]: Good point. Let me try it out.

File: scripts/test.sh (hunk 1 above)
Comment (tonyyang-svail) [reply scripts/test.sh:6]: You are right. I forgot the tests in `cmd/` also need the populated data. My bad.

**Comment by tonyyang-svail** [comment]
Follow @typhoonzero 's suggestion, using `set -e` directly in `travis.yml`. PR submitted in https://github.com/sql-machine-learning/sqlflow/pull/381 .

**Pull request closed without merging**